logstash-core/lib/logstash/runner.rb

```diff
@@ -90,6 +90,10 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.web_api.flag.http_port"),
     :attribute_name => :web_api_http_port, :default => 9600
 
+  option ["--[no-]log-in-json"], :flag,
```

Comment:
Thoughts about renaming this flag to `--logger.format` to make it future proof for `setting.yml`? This way it translates easily to yml.. See https://github.com/elastic/logstash/pull/4499 /cc @jsvd